lib/vanagon/platform.rb

```diff
@@ -226,7 +230,7 @@ def is_unix?
     #
     # @return [true, false] true if it is a windows variety, false otherwise
     def is_windows?
-      return !!@name.match(/^win-.*$/)
+      return !!@name.match(/^windows-.*$/)
```

Comment:
Like I said, mixed feelings. But 2007 was 9 years ago; I probably wouldn't cite anything Ruby related from 9 years ago. Note that I do not totally disagree with you, though after thinking about it I find the intent in `!thing.nil?` a little easier to parse at a glance.